vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderGroup.kt

```diff
@@ -154,7 +154,8 @@ class RoomListSectionBuilderGroup(
         ) {
             it.memberships = listOf(Membership.JOIN)
             it.roomCategoryFilter = RoomCategoryFilter.ALL
-            it.roomTagQueryFilter = RoomTagQueryFilter(false, false, false)
+            // Tchap: Show low priorities in room list
+            it.roomTagQueryFilter = RoomTagQueryFilter(false, null, false)
```

Comment:
maybe you can do:
`
it.roomTagQueryFilter = RoomTagQueryFilter(isFavorite = false, isLowPriority = null, isServerNotice = false)
`
i think it will be more understandable

Reply:
I agree but I prefer to stay aligned with Element: https://github.com/vector-im/element-android/blob/ab29a155724b80e8fbc32c034a2bbde8c1ee3d32/vector/src/main/java/im/vector/app/features/home/room/list/RoomListSectionBuilderGroup.kt#L157